test/connect.js

```diff
@@ -29,7 +29,7 @@ test('map connect address eid for valid IPC address', function (t) {
   }
 
   sock.close();
-  require('fs').unlink('some_address');
+  require('fs').unlinkSync('some_address');
```

Comment:
good call...

btw what a shame nodejs has done away with **async fire-and-forgets**

Reply:
also I have a feeling this might not need to be unlinked after the update... IPC sockets ought to clean up after themselves, and if not then either libnanomsg or node-nanomsg should probably do that. Imagine if you had to delete TCP ports after using them. 

Right.. I can't remember whether OSX or Linux clears out that descriptor afterwards, but I think at least one of them leaves it behind, 

which means `unlink()` might do like an `ENOENT` or something (perfect use case for **async fire-and-forget**)